AWSutils.sh

```diff
@@ -385,6 +385,17 @@ function InstallCfnBootstrap {
    fi
 }
 
+# shellcheck disable=SC2016,SC1003
+function ProfileSetupAwsCli {
```

Comment:
Funny thing is, aws-cli v2 does this automatically already (set the default region from instance metadata). What doesn't do it are all the SDKs. So this is for every tool that uses an AWS SDK _except_ the aws-cli v2.